packages/dev/test/StoredTransaction.test.ts

```diff
@@ -33,7 +32,7 @@ contract('StoredTransaction', function (accounts) {
       data: '0x12345678',
       nonce: 111,
       txId: '0x437a03ff976c96807aad1bc895f1535292e4ddf798fd8232600d89912961fc57',
-      value: bufferToHex(toBuffer(1e18))
+      value: `0x0${1e18.toString(16)}`
```

Comment:
ugly. we use `import { toHex } from 'web3-utils'` all over..

Reply:
Unfortunately, `toHex` won't work here.
The objects are later compared using `assert.deepEqual`, and transactions are created using buffer.toHex, which pads the value with a leading zero. This won't happen for any normal int-to-hex conversion, so I had to do a manual change. 
Note that this issue is not related to this PR per se. It is a result of a fresh `yarn install`.

Reply:
Why did you remove the imports in the first place?